### PR TITLE
Updating contributing.md to make it more clear. Fixes Issue #107

### PR DESCRIPTION
**Pull request by Ethan-bradley**
Fixes Issue #107. This makes the contributing doc more clear and fixes some bugs.
Changes listed below:
Maybe add that estimated time to setup everything (including installing docker) is around 30-45 minutes.
Add Open terminal
Add Install git (May add instructions for git installation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html) if not already installed. If you are on a mac, just type in git and hit ok to install software tools.
Run the following commands in terminal:
Change git clone line to git clone https://github.com/openoakland/woeip.git, currently incorrect.
Separated the first two code lines to allow for easier copy and paste.
That make local.shell shoud look like this (outside of shell):
make local.shell
Add go to 0.0.0.0:8080 on your web browser
Put in the username that you put in the terminal for the email.

---
name: Pull Request
about: Request merging changes into the code base
title: ''
labels: ''
reviewers: ''

---

## Checklist
- [ ] Add description
- [ ] Reference open issue pull request addresses
- [ ] Pass linting check
  - complete on the local machine with `make local.shell` `make quality`
- [ ] Pass functional tests
  - complete on the local machine with `make local.shell` `make test`
- [ ] Request code review
  - Please allow **36 hours** from opening a pull request before merging a pull request- even if it has already recieved an approving review.
- [ ] Address comments on code and resolve requested changes
- [ ] Merge own code

## Description
Issue: #

*Brief description of solution*

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -24,11 +24,17 @@ These are general guidelines on the flow from design to implementation
 5. Github pull requests implement the code agreed upon in the GitHub issues
 
 ## Code development
+Following the installation steps (including installing docker) takes around 20-35 minutes. 
 
+Make sure you have git installed (Instructions for git installation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html if not already installed).
```
Comment (TangoYankee) [inline .github/contributing.md]: Recommend Markdown link for styling: `([Instructions for git installation](https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html))`

File: .github/contributing.md [hunk 2]
```diff
@@ -50,15 +56,20 @@ When running the service with this command, it will be configured to run using t
...
+```bash
+make local.shell
 ```
 
 Create a Django superuser:
 
 ```
 root@<container ID>:/app/woeip#	./manage.py createsuperuser 
 ```
+Enter a username, password and email. These are for logging into the docker container.
```
Comment (TangoYankee) [inline .github/contributing.md]: The superuser is for the Django Admin

Comment (Ethan-bradley) [reply .github/contributing.md]: will add

File: .github/contributing.md (hunk 1 above)
Comment (Ethan-bradley) [reply .github/contributing.md]: will change the style

**Review by theecrit — APPROVED** [review]

File: .github/contributing.md (hunk 2 above)
Comment (theecrit) [reply .github/contributing.md]: I committed the change directly to your branch; hope that's okay! I figured it was a minor enough change.

**Comment by Ethan-bradley** [comment]
That is probably fine

**Pull request merged**